a3fe/run/simulation.py

```diff
@@ -526,7 +555,6 @@ def read_gradients(
     def update_paths(self, old_sub_path: str, new_sub_path: str) -> None:
```

Comment:
If the only thing done is to call super, there's no need to include the function.